Gopkg.lock

```diff
@@ -132,12 +132,12 @@
   version = "v1.0.0"
 
 [[projects]]
-  digest = "1:3f53e9e4dfbb664cd62940c9c4b65a2171c66acd0b7621a1a6b8e78513525a52"
+  digest = "1:87c2e02fb01c27060ccc5ba7c5a407cc91147726f8f40b70cceeedbc52b1f3a8"
   name = "github.com/sirupsen/logrus"
   packages = ["."]
   pruneopts = "UT"
-  revision = "ad15b42461921f1fb3529b058c6786c6a45d5162"
-  version = "v1.1.1"
+  revision = "e1e72e9de974bd926e5c56f83753fba2df402ce5"
+  version = "v1.3.0"
```

Comment:
I considered this. Perhaps I'm wrong, but I thought specifying v1.2.0, even though it locked at v1.3.0, would allow projects using versions as old as v1.2.0 to import this without having to override in dep.

Reply:
Ok, lets stick with this approach.